feature-libs/cart/base/components/cart-shared/cart-item-list-row/cart-item-list-row.component.html

```diff
@@ -10,6 +10,7 @@
     <div class="cx-table-item-container">
       <!-- Item Image -->
       <a
+        *ngIf="!options.disableItemLink"
```

Comment:
see below comments about introduction of new 'disableItemLink' toggle

Reply:
Not a breaking change in my opinion. The toggle is optional,  DOM stays unchanged for existing customer not using this toggle.